pages/robots/rbts-myplanet.md

```diff
@@ -17,8 +17,8 @@ To get started vist Open Learing Exchange on GitHub to view all working
 [repositories](https://github.com/open-learning-exchange). Or jump straight to the [myPlanet](https://github.com/open-learning-exchange/myplanet) 
 repository's README file where you can find set up information. 
 
-## Usefull Links
```

Comment:
This should be separate PR so please clean this PR just to put your profile file and create separate PR for this issue